usagov/vote-gov-drupal
# WIP - VOTE-164: Disable the contrib module, block_content_permissions.

**NOTE: Do not merge until PR https://github.com/usagov/vote-gov-drupal/pull/358 is merged into Dev. Once complete, merge dev into this branch to resolve any possible conflicts and to capture the new block's permissions.**



## Jira ticket

[VOTE-164](https://cm-jira.usa.gov/browse/VOTE-164)

## Description

Disable and remove the block_content_permissions contributed module. This functionality is now within Drupal Core.


## Deployment and testing

### Pre-deploy

1. Uninstall Block content permissions module

### Post-deploy

1. Locally, run `lando drush cim` to import changes.

### QA/Test

1. Login as an admin.
2. Visit the user permissions page and confirm that block permissions are set appropriately for Content Manager.

## Checklist for the Developer




## Checklist for the Peer Reviewers
